schemas/v1.0/azure.yaml.json

```diff
@@ -27,6 +27,31 @@
                 }
             }
         },
+        "infra": {
+            "type": "object",
+            "additionalProperties": true,
+            "properties": {
+                "provider":{
+                    "type": "string",
+                    "title": "Type of infrastructure provisioning provider",
+                    "description": "Optional. When specified will set the infrastructure provisioning provider used to provision the Azure resources for the application. (Default: bicep)",
+                    "enum": [
+                        "bicep",
+                        "terraform"
+                    ]
+                },
+                "path": {
```

Comment:
Maybe the same question can be imposed: do we think this is a useful enough feature to break convention? Being opinionated about repo structure might not be a bad thing.

Reply:
I believe there was an feature request issue filed for this.  Will search for it.